protected/humhub/modules/comment/Module.php

```diff
@@ -76,11 +76,6 @@ public function canComment($object)
             return false;
         }
 
-        // Only allow one level of subcomments
-        if ($object instanceof Comment && $object->object_model === Comment::class) {
```

Comment:
I think its not a good idea to remove this check here, since this would allow multi level sub comments on controller level. 

In case of a sub comment we should check if the parent object is commentable in the view:

If object is not commentable by `canComment` check if its a sub comment and if the user can can comment the parent comment and then render special reply.